nemo_aligner/algorithms/ppo.py

```diff
@@ -569,9 +554,10 @@ def fit(self):
                 )
 
                 if run_val:
-                    self.timer.start("validation_time")
-                    val_metrics = self.run_validation()
-                    timing_metrics["validation_time"] = self.timer.stop_and_get_time("validation_time")
+                    with self.timer("validation_time"):
+                        val_metrics = self.run_validation()
+                    # TODO(terryk): is it okay to log validation_time one step behind?
```

Comment:
oh i see, yes there is an off by 1 issue with val. i think that's fine, val at step 5 means we've completed step 4

Reply:
changed TODO to NOTE in 73f737180aeb5d4afc72140fc66f33d1544693b0